govc/test/vcsim.bats

```diff
@@ -301,7 +301,7 @@ docker_name() {
   assert_success
 
   ip=$(govc object.collect -s vm/$vm guest.ipAddress)
-  run docker run --rm curlimages/curl curl -f "http://$ip/vcsim.bats"
+  run curl -f "http://$ip/vcsim.bats"
```

Comment:
I'm using "Docker Desktop 3.2.2 (61853)".

The `powerclicore` image has `curl` installed, how about just switching from curlimages/curl to powerclicore?:
Or just leave as-is and we can sort it out later, CI is more important :)

Can you install the gnu tools to avoid the xargs errors locally?
https://github.com/vmware/govmomi/blob/master/govc/test/README.md#darwin-macosx

Reply:
Thx for the hint, installed the tools. Yeah, let's see if local `curl` works otherwise I'll revert back.